test/es-module/test-typescript-commonjs.mjs

```diff
@@ -59,12 +59,24 @@ test('require a .ts file with implicit extension fails', async () => {
 });
 
 test('expect failure of an .mts file with CommonJS syntax', async () => {
-  const result = await spawnPromisified(process.execPath, [
-    fixtures.path('typescript/cts/test-cts-but-module-syntax.cts'),
-  ]);
+  const testFilePath = fixtures.path('typescript/cts/test-cts-but-module-syntax.cts');
+  const result = await spawnPromisified(process.execPath, [testFilePath]);
+
+  assert.strictEqual(result.stdout, '');
+
+  const expectedWarning = `Failed to load the ES module: ${testFilePath}. Make sure to set "type": "module" in the nearest package.json file or use the .mjs extension.`;
+  try {
+
+    assert.ok(result.stderr.includes(expectedWarning), `stderr does not contain the expected warning. Actual stderr: ${result.stderr}`);
```

Comment:
FWIW you don't really need to supply a custom message given that we already override the error properties to add the useful information there (but having the custom message is also fine if you prefer to keep it)
```suggestion
    assert.ok(result.stderr.includes(expectedWarning));
```

Reply:
thanks